protocol-designer/src/components/organisms/AssignLiquidsModal/LiquidToolbox.tsx

```diff
@@ -273,12 +278,35 @@ function LiquidToolbox({
       : false
   const hasSelection = selectedWells.length > 0
   const canEdit = !showLiquidLayoutOverlay && (hasLiquids || hasSelection)
+  const { labware, modules } = useSelector(getInitialDeckSetup)
+  const stackerModuleIds = Object.entries(modules).reduce<string[]>(
+    (result, [key, value]) => {
+      return value.type === FLEX_STACKER_MODULE_TYPE
+        ? result.concat(key)
+        : result
+    },
+    []
+  )
 
+  const labwareOnStackerWithLiquid = Object.entries(labware).filter(
+    ([, labwareDef]) =>
+      labwareDef.stack != null &&
+      stackerModuleIds.some(stacker => labwareDef.stack.includes(stacker))
```

Comment:
small nit to make this clearer 

```suggestion
      stackerModuleIds.some(stackerId => labwareDef.stack.includes(stackerId))
```